rent.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :books
+
+  def initialize(date, person, books)
+    @date = date
+    @person = person
+    @book = books
+    books.rentals << self
+    person.rentals << self
+  end
+end
```

Comment:
a method for creating rentals must be present in the person.rb file and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`👍🏾